odoo/orm/fields_properties.py

```diff
@@ -717,10 +717,13 @@ def condition_to_sql(self, field_expr: str, operator: str, value, model: BaseMod
         if isinstance(value, str):
             sql_left = SQL("(%s ->> %s)", raw_sql_field, property_name)  # JSONified value
             sql_right = SQL("%s", value)
-            return SQL(
+            sql = SQL(
                 "%s%s%s",
                 unaccent(sql_left), sql_operator, unaccent(sql_right),
             )
+            if Domain.is_negative_operator(operator):
+                sql = SQL("(%s OR %s IS NULL)", sql, sql_left)
```

Comment:
What if `value` is empty (`(..., 'not like', '')`) ?  Is there a test case for this one?